packages/legend-application/src/components/ValueSpecificationEditor.tsx

```diff
@@ -14,54 +14,54 @@
  * limitations under the License.
```

Comment:
`ValueSpecificationEditor` sounds too broad for me because this really only deal with basic stuff like instance value spec, I leave the rename to you, but I would prefer something like `Basic/SimpleValueSpecificationEditor`